### PR TITLE
DOC: Fix documentation typos

### PR DESCRIPTION
**Pull request by jhlegarreta**
- DOC: Fix typos in method docstrings
- DOC: Fix method parameter name in docstring

### CODE REVIEW

File: dipy/align/expectmax.pyx [hunk 1]
```diff
@@ -26,7 +26,7 @@ def quantize_positive_2d(floating[:, :] v, int num_levels):
     The following are undefined, and raise a ValueError:
     * Quantizing at zero levels because at least one level must be assigned
     * Quantizing at one level because positive values should be assigned a
-      level different from the secial level 0 (at least 2 levels are needed)
+      level different from the special level 0 (at least 2 levels are needed)
```
Comment (jhlegarreta) [inline dipy/align/expectmax.pyx:29]: Assumed it was special, but not sure.

File: dipy/align/expectmax.pyx [hunk 2]
```diff
@@ -125,7 +125,7 @@ def quantize_positive_3d(floating[:, :, :] v, int num_levels):
...
```
Comment (jhlegarreta) [inline dipy/align/expectmax.pyx:128]: Same as above.

**Comment by jhlegarreta** [comment]
Build failures are unrelated.

**Comment by skoudoro** [comment]
Thank you @jhlegarreta! merging!

**Pull request merged**